Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -481,7 +481,10 @@ void CGameListCtrl::InsertItemInReportView(long _Index)
 		}
 	}
 
-	SetItem(_Index, COLUMN_TITLE, StrToWxStr(name), -1);
+	if (rISOFile.IsDiscTwo() && !name.find("DISC TWO", 0))
```

Comment:
Unsure, I only have one multi-disc game which uses DISC TWO. I'd like confirmation to clear out the majority of edge cases